app/src/debug/java/org/mifos/mobile/api/BaseURL.kt

```diff
@@ -11,13 +11,19 @@ class BaseURL {
     val defaultBaseUrl: String
         get() = PROTOCOL_HTTPS + API_ENDPOINT
 
+    val paymentUrl: String
+        get() = PROTOCOL_HTTP + PAYMENTAPI_ENDPOINT + PAYMENT_API_PATH
```

Comment:
Are we using `Http` or `Https`?

Reply:
The APIs of `PaymentHub` use the `Http` protocol, while the other APIs of this project uses `Https`. Therefore we are using both of them.